tutorials/docker-gcplogs-driver/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using the Docker gcplogs Logging Driver
+description: Learn to use the gcplogs logging driver to save your Docker logs to Stackdriver Logging.
+author: tswast
+tags: Docker, Logging
+date_published: 2017-04-24
+---
+This tutorial will show you how to use the [Docker gcplogs logging
+driver](https://docs.docker.com/engine/admin/logging/gcplogs/) to write your
+Docker logs to [Stackdriver Logging](https://cloud.google.com/logging/).
+
+## Objectives
+
+- Run a Docker container with the gcplogs logging driver.
+- View logs in the Google Cloud Platform console.
+
+## Before you begin
+
+1.  Create or select a Cloud Platform project from the [Google Cloud Platform
+    console's projects page](https://console.cloud.google.com/project).
+1.  [Enable
+    billing](https://support.google.com/cloud/answer/6293499#enable-billing)
+    for your project.
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform including
+
+- [Google Compute Engine](https://cloud.google.com/compute/pricing)
+- [Stackdriver Logging](https://cloud.google.com/stackdriver/pricing)
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator/) to
+estimate the costs for your usage.
+
+## Setting up the virtual machine
+
+Create a new Compute Engine instance using the [CoreOS](https://coreos.com/why)
```

Comment:
Because it wasn't in the OS list two weeks ago when I started looking at how to do this. :-) Updated.